VMS/README.VMS

```diff
@@ -1,5 +1,13 @@
 Dear OpenVMS user,
 
+**Important**
+
+We will drop support to VMS in version v2.4.0. We never test on VMS,
+and we have no way of doing so. And we haven't had any report on the
+topic in years. Pleas feedback if you still need the support.
```

Comment:
```suggestion
topic in years. Please provide feedback if you still need the support.
```

Reply:
Thanks you. More suggestions are welcome. :)